app/src/main/java/de/westnordost/streetcomplete/quests/leaf_detail/AddTreeLeafType.kt

```diff
@@ -0,0 +1,37 @@
+package de.westnordost.streetcomplete.quests.leaf_detail
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.elementfilter.toElementFilterExpression
+import de.westnordost.streetcomplete.data.osm.geometry.ElementGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.OUTDOORS
+import de.westnordost.streetcomplete.osm.Tags
+
+class AddTreeLeafType : OsmFilterQuestType<TreeLeafType>() {
+    override val elementFilter = """
+        nodes with natural = tree and !leaf_type
+        and !species:wikidata and !species:wikipedia and !genus:wikidata and !genus:wikipedia
+        and !genus and !~genus:.* and !species and !~species:.* and !taxon and !~taxon:.*
+        and !taxon:species and !~taxon:species:.* and !taxon:genus and !~taxon:genus:.*
```

Comment:
Some reordering to make it more readable:

```suggestion
        and !species and !species:wikidata and !species:wikipedia and !~species:.*
        and !genus and !genus:wikidata and !genus:wikipedia and !~genus:.*
        and !taxon and !~taxon:.*
        and !taxon:species and !~taxon:species:.*
        and !taxon:genus and !~taxon:genus:.*
```